io/src/save.rs

```diff
@@ -13,7 +13,7 @@ const WRITE_ERROR: &str = "Error writing file: ";
 
 /// Serializable save data.
 #[derive(Deserialize, Serialize)]
-pub(crate) struct Save {
+pub struct Save {
```

Comment:
This should be reverted. Ideally, main.rs parses the cli args and then calls a function in io which loads the save file. I don't want to expose the save file to the other crates.